_pages/teaching.markdown

```diff
@@ -10,7 +10,49 @@ toc_sticky: true
 
 
 header:
-  image: /assets/images/wip.jpg
-  caption: "Photo credit: [**Gaelle Marcel**](https://unsplash.com/@gaellemarcel?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText) on [**Unsplash**](http://unsplash.com/)"
+  image: /assets/images/teaching/teaching.jpg
+  caption: "Photo credit: [**Diego PH**](https://unsplash.com/@jdiegoph?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText) on [**Unsplash**](http://unsplash.com/)"
 
 ---
+
+KBSS group main teaching scope is in Semantic Web, Ontologies and Enterprise architectures in Java. We also participate in teaching database systems.
```

Comment:
Semantic Web and Enterprise architectures are the main focus of our group that reflects in our offered topics for bachelor and master thesis [thesis section link] and our main teaching activities [link to paragraphs about EAR paragraph link, OSW paragraph link]. In addition, we participate in the teaching of _Software Architectures_ [SWA section link] and _Database systems_ [DBS paragraph link].

Reply:
Consider using some ideas that I put here.

- Moje motivate: 
  - chtel bych separovat veci, u kterych jsme garanti a ktere jenom dopomahame (verim ze ty nase jsou lepsi kvality :)). 
  - Ontologie jsem vyhodil, protoze ty stejne nejsou pojem, ktery jim neco rovnou rekne ... Je to jeste mene zname jako Semanticky Web. Rozhodne nam chyba jeste stranka ohledne research, kde bychom ty vsechny techonolgie pospojovali a udelali to atraktivni tim, ze ukazeme ze to jsou velmi web-friendly technologie, ktere budu v budoucnosti pouzivane ... jestli se to bude nazyvat Semantic Web/Ontologie nebo uplne jinak ... nato jsem delal i ticket #10 #8. Takze jestli neco takoveho budem mit tak to muzem prolinkovat.

Reply:
ok